lib/hpcsocket.js

```diff
@@ -114,8 +121,8 @@ HPCSocket.prototype.handleError = function (data) {
 HPCSocket.prototype.connectionError = function (err) {
 
   var self = this;
-  
-  err = "Problem connecting to " + setup.cluster_ip + ". " + err;
+
+  err = "Problem connecting to " + default_setup_url + ". " + err;
```

Comment:
I thought we were going to assign the url to an attribute within HPCSocket and then reference self.url here.